<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sellDialog</class>
 <widget class="QDialog" name="sellDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>521</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/middle.jpg</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabWidgetPage1">
       <attribute name="title">
        <string>销售信息查询</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>681</width>
          <height>371</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>商品条码</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>供货商</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>销售日期</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="s_id"/>
           </item>
           <item>
            <widget class="QPushButton" name="search">
             <property name="text">
              <string>查询</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="tableView"/>
         </item>
        </layout>
       </widget>
       <widget class="QToolButton" name="toolButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>390</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string> 导出到 Excel</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabWidgetPage2">
       <attribute name="title">
        <string>销售清单写入</string>
       </attribute>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>671</width>
          <height>341</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_3">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>0</y>
          <width>261</width>
          <height>30</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="Label_11">
           <property name="text">
            <string>商品条码</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="in_code"/>
         </item>
         <item>
          <widget class="QLabel" name="Label_15">
           <property name="text">
            <string>商品数量</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="in_qty"/>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>0</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>390</y>
          <width>271</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>总价：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="total_pay">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="submit">
           <property name="text">
            <string>确认写入</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delete_item">
           <property name="text">
            <string>删除该条</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>390</y>
          <width>316</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>会员ID：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="vip_id"/>
         </item>
         <item>
          <widget class="QPushButton" name="vip_login">
           <property name="text">
            <string>会员登陆</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="vip_logoff">
           <property name="text">
            <string>会员注销</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
